var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -45,7 +45,7 @@ class Sensei(CMakePackage):
     variant('python', default=False, description='Enables Python bindings')
     variant('miniapps', default=True, description='Enable the parallel 3D and oscillators miniapp')
 
-    depends_on("paraview@5.5.0:5.6.0", when="+catalyst")
+    depends_on("paraview@5.5:5.6", when="+catalyst")
```

Comment:
The version change here is super useful but isn't really well documented.  Generally when expressing versions in dependencies, spack considers any missing version component to be a wildcard.  So by specifying `5.5:5.6` what that actually translates to is `5.5.0:5.6.99999`